packages/angular_devkit/core/src/workspace/workspace.ts

```diff
@@ -140,6 +140,7 @@ export class Workspace {
     return workspaceProjectClone;
   }
 
+  // `defaultProject` is deprecated and should be removed in CLI v8.
   getDefaultProjectName(): string | null {
```

Comment:
Can you use `@deprecated`? As when using it tslint is able to warn users. Same thing in the IDE

https://palantir.github.io//tslint/rules/deprecation/